src/views/admin/PersonManagementView.vue

```diff
@@ -64,7 +64,7 @@
   );
 
   const schulen: ComputedRef<TranslatedObject[] | undefined> = computed(() => {
-    return organisationStore.allSchulen
+    return organisationStore.allOrganisationen
```

Comment:
In der Liste sollen also nicht nur Schulen, sondern unterschiedliche Organisationen sein? Dann bitte auch die Variablen entsprechend durchgehend benennen, nicht manchmal Schule und manchmal Organisation.

Reply:
Generell ist es nicht schön, dass wir manchmal in der UI Schule nutzen und manchmal Orga aber so waren Anforderungen. Es waren ja nur Schulen hier vorher aber jetzt sollten laut PO auch Orgas vom Typ "Land" erscheinen. Allerdings heissen die im Filter immernoch "Schulen"... Ich ändere das jetzt trotzdem im Code.

Reply:
Christian_: Moin Youssef, danke für den Vorschlag. Lass es aber bitte bei Schule. 98% der Anwender sind Schuladmins, die kennen Schulen. Die Landesadmins können nach eigener Aussage hier differenzieren._

Ich habe jetzt die Variablen von Schule zu Orga aber UI bleibt scheinbar bei Schule.